src/library/VTKExtensions/CMakeLists.txt

```diff
@@ -0,0 +1,65 @@
+# XXX not sure this is the right location
+# this is at least the deepest possible
+
+# Provide symbolic link to data in build
+# Required for testing
+if(BUILD_TESTING)
+  file(CREATE_LINK "${CMAKE_SOURCE_DIR}/testing/" "${CMAKE_CURRENT_BINARY_DIR}/Data" SYMBOLIC)
```

Comment:
I don't like that

Reply:
me neither, I think Ill just open a VTK issue to improve that and use a custom solution for now.

Reply:
ok sounds good

Reply:
https://gitlab.kitware.com/vtk/vtk/-/issues/18482

Reply:
note it can be done like this:

```
set_tests_properties(f3d::VTKExtensionsRenderingCxx-TestF3DRenderer PROPERTIES ENVIRONMENT "VTK_BASELINE_ROOT=${CMAKE_SOURCE_DIR}/testing/baselines;VTK_DATA_ROOT=${CMAKE_SOURCE_DIR}/testing/data;VTK_TEMP_DIR==${CMAKE_BINARY_DIR}/Testing/Temporary")
```

But I'd rather not do that.